packages/pyright-internal/src/analyzer/service.ts

```diff
@@ -318,7 +317,7 @@ export class AnalyzerService {
     updateOpenFileContents(
         path: string,
         version: number | null,
-        contents: TextDocumentContentChangeEvent[],
```

Comment:
@heejaechang I believe this would mean we'd have to do updates for say fixall differently. Maybe other spots too.

Reply:
I think this works fine for the current fixall strategy. These changes are applied only to source files associated with a cloned program.

Reply:
Oh it looks like fixall uses something you already fixed. the workspaceEditUtils.applyDocumentChanges.